pubspec.yaml

```diff
@@ -15,15 +15,8 @@ dependencies:
 
 dev_dependencies:
   build: ^0.12.0
-  build_runner: ^0.7.9
+  build_runner: '>=0.7.9 <0.9.0'
```

Comment:
it's a dev dependency and we're already restricted to dart 2 - why do we need to support old build_runner versions?

Reply:
Was trying to just support latest. No issues bumping to `^0.8.0`